common/gtk-3.0/3.20/sass/_common.scss

```diff
@@ -1601,6 +1601,7 @@ menubar,
 menu,
 .menu {
   $_menu_bg: if($variant=='light', $base_color, $bg_color);
+  $_sp_menu_bg: #dfdfdf;
```

Comment:
This should get a color from the predefined colors in colors.scss and not a fixed value like this. It also have to work with the dark theme.

Reply:
Yes I know it needs more work, but I think you guys will improve it.